build.sbt

```diff
@@ -92,6 +93,7 @@ lazy val allOld = (project in file("allOld"))
       Dependencies.scalatestSelenium.value ++
       Dependencies.moduleMysql.value ++
       Dependencies.modulePostgres.value ++
+      Dependencies.moduleOracle.value ++
```

Comment:
You forgot about this line :) `allOld` should not be changed.

Reply:
removed!